modules/ref_fb_module/src/statistics_fb_impl.cpp

```diff
@@ -244,15 +281,19 @@ void StatisticsFbImpl::processSignalDescriptorChanged(const DataDescriptorPtr& v
     configure();
 }
 
-void StatisticsFbImpl::processDataPacket(const DataPacketPtr& packet)
+void StatisticsFbImpl::processDataPacketTrigger(const DataPacketPtr& packet)
 {
-    if (!valid)
-        return;
-
     const auto domainPacket = packet.getDomainPacket();
-    if (!domainPacket.assigned())
-        return;
```

Comment:
Is this check no longer required?

Reply:
It was in the original statistics code, so I'm not sure. Can we assume that `domainPacket` is always assigned?

Reply:
Well, in a way we can, but it's just an extra guard against malformed inputs, so it doesn't hurt for it to be there.